includes/express-checkout/class-wc-payments-express-checkout-ajax-handler.php

```diff
@@ -196,6 +196,9 @@ public function ajax_get_shipping_options() {
 	public function ajax_get_cart_details() {
 		check_ajax_referer( 'wcpay-get-cart-details', 'security' );
 
+		// Process added gift cards.
+		add_filter( 'woocommerce_gc_disable_ui', '__return_false' );
```

Comment:
While working on https://github.com/Automattic/woocommerce-payments/pull/9846 I noticed the express checkout buttons (Apple/Google Pay) were not getting hidden on classic checkout when the cart total get to 0 by adding gift cards, as the PR was closed I'm taking the opportunity of this PR to add this line and fix that.

Reply:
Can you provide test instructions?

Reply:
@hsingyuc Testing instructions:

* Install Gift Cards.
* Create new Gift Card codes.
* Expose your site with Jurassic Tube or ngrok to enable Google/Apple Pay buttons.
* Add products to the cart.
* Access the classic checkout page.
* Add gift card codes until the total cart value reach 0.
* The Google/Apple Pay buttons should be hidden.